gitcoinco/grants-stack
# Create ipfs collections 2

part of #3299

This PR adds a button to export the cart to a sharable collection saving the details into a json file saved on IPFS.

The first 2 commits  fix some invalid html and react errors.

The commit `fix isAlreadyInCart functionalities` fixes the way we check if an application is already in the cart.

### How to test it

* Open the preview link.
* Add some applications to the cart.
* Go to the cart page
* You should see this new button:
  <img width="817" alt="Screenshot 2024-04-18 at 03 09 12" src="https://github.com/gitcoinco/grants-stack/assets/1059/6d0c779b-2a15-4a13-8733-f71fd0d492e4">
* Click on it and create your collection
  <img width="570" alt="Screenshot 2024-04-18 at 03 09 23" src="https://github.com/gitcoinco/grants-stack/assets/1059/caaef54b-fe50-400b-85a0-5b4e7e98766e">
* The last step will generate a unique URL for the collection
  <img width="534" alt="Screenshot 2024-04-18 at 03 09 32" src="https://github.com/gitcoinco/grants-stack/assets/1059/3da6fe7e-748e-4ae5-940e-f55bbf381335">
